client/app/libs/slim_lodash.js

```diff
@@ -0,0 +1,3 @@
+import bindAll from 'lodash/bindAll';
+import find from 'lodash/find';
+export default { bindAll, find };
```

Comment:
Thanks for clarifying @megawac.

@justin808 @alexfedoseev I'm going to close this PR. I'm not the right person to try to implement this functionality, I don't have enough experience with webpack/babel/es6 modules, and I think my efforts are just causing more confusion and wasting reviewers' time. I suspect it will be easy for someone else with more experience.
cheers